poc_iot_injector/src/server.rs

```diff
@@ -100,9 +107,10 @@ async fn submit_txns(
     txn_service: &mut TransactionService,
     store: &mut FileStore,
     keypair: Arc<Keypair>,
+    receipt_sender: &file_sink::MessageSender,
     after_utc: DateTime<Utc>,
     before_utc: DateTime<Utc>,
-) -> Result {
+) -> Result<()> {
```

Comment:
isn't this covered by the definition of result in the error module?
```
pub type Result<T = ()> = std::result::Result<T, Error>;
```

Reply:
Oh yeah, missed it. I'll update.